Protobuf/system/object.proto

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+syntax = "proto3";
+
+package dolittle.events.relativity;
+
+
+option csharp_namespace = "System.Protobuf";
+
+// Represents the artifact
```

Comment:
Both object.proto and value.proto "Represents the artifact"

Reply:
I can't seem to follow

Reply:
Good catch - both files will be gone in a future commit in a completely different pull request on a different branch :)